src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedWorkspaceSymbolFeature.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+package com.redhat.devtools.lsp4ij.server.definition.launching;
+
+import com.redhat.devtools.lsp4ij.client.features.LSPWorkspaceSymbolFeature;
+
+/**
+ * Adds client-side workspace symbol configuration features.
+ */
+public class UserDefinedWorkspaceSymbolFeature extends LSPWorkspaceSymbolFeature {
+
+    @Override
+    public boolean supportsGotoClass() {
```

Comment:
Following the same pattern as before where the implementation that consults client config is in the `UserDefined*` subclass, and it's added in the same manner.